packages/file_selector/file_selector_android/android/src/main/java/dev/flutter/packages/file_selector_android/FileUtils.java

```diff
@@ -206,4 +212,27 @@ private static String getBaseName(String fileName) {
     // Basename is everything before the last '.'.
     return fileName.substring(0, lastDotIndex);
   }
+
+  protected static String sanitizeFilename(String displayName) {
+    if (displayName == null) {
+      return null;
+    }
+
+    String[] badCharacters = new String[] { "..", "/" };
+    String[] segments = displayName.split("/");
+    String fileName = segments[segments.length - 1];
+    for (String suspString : badCharacters) {
```

Comment:
I'm confused as to why the docs do this step since it can't possibly contain any `/`s at this point, but 🤷🏻

Reply:
Yeah... I'm inclined to leave it because its written by the privacy and security folks at Android but it's definitely odd...

Reply:
I prefer to leave this code as it exists in the android documentation.

Reply:
Yes, that's fine with me.